### DISCUSSION TITLE
issue with CRON setup

### DISCUSSION THREAD

**Discussion by harora4** [kedacore/keda#3279] [General]
Hi Team,

We are setting up keda cron to bring the deploymen replica to 0 and then scale to 3 at a particular timestamp.
Issue :- The moment we push the scaleobject yaml, the HPA triggers to bring the replicas to 0 immediately rather triggering at certain time we specified and is not working the desired manner.
Please guide us on the same

We have test deployment setup using nginx with 3 replicas.
$kubectl get all -n <namespace>
NAME                           READY   UP-TO-DATE   AVAILABLE   
deployment.apps/test   0/0            0            0           
NAME                                      DESIRED   CURRENT   READY   
replicaset.apps/test-5f6778868d   0         0         0       
NAME                                                                     REFERENCE         TARGETS                                MINPODS   MAXPODS   REPLICAS  
horizontalpodautoscaler.autoscaling/keda-hpa-cron-scaledobject   Deployment/test   <unknown>/1 (avg), <unknown>/1 (avg)   1         100       0         

Below yaml being used :-
---
apiVersion: keda.sh/v1alpha1
kind: ScaledObject
metadata:
  name: cron-scaledobject
  namespace: test
spec:
  scaleTargetRef:
    kind: Deployment
    name: test
  pollingInterval: 30
  triggers:
  - type: cron
    metadata:
      timezone: America/Edmonton
      start: 23 * * * *
      end:  27 * * * *
      desiredReplicas: "0"
  - type: cron
    metadata:
      timezone: America/Edmonton
      start: 37 * * * *
      end:  40 * * * *
      desiredReplicas: "3"

**Comment by JorTurFer** [comment]
Hey,
Sorry for the delay response. It seems correct, could you share any log?
